Added Professor Class

YuvalDeshe opened the pull request:
Added the Professor class to be used in webscraping; #21

Jeffrey-000 commented:
some issues:
-create class constructors
-change the class property visibility to private; currently is package protected
-create necessary getters and setters
-specifically line 11 note: setting the values of the array should be done by a constructor and after the variables have be initialized

YuvalDeshe commented:
The commit 07510e0:
 - Added 3 constructors.
 - Made all variables within the class private.
 - Created a getter and a setter method for each variable (two setters for ratingDistribution).
 - Added documentation for each method.

Jeffrey-000 commented on backend/webscraping/Professor.java:
```diff
@@ -0,0 +1,389 @@
+    public Professor(String name, double quality, double wouldTakeAgainPercent, double difficulty, 
+                     int numOneStar, int numTwoStar, int numThreeStar, int numFourStar, 
+                     int numFiveStar, int numRatings, String[] tags, String department) {
+        name = this.name;
```
the variables are in the wrong order
should be --> this.name = name;

YuvalDeshe commented:
The commit 40de972 fixed the order of the variables.

YuvalDeshe commented:
The commits 1de790b and b53cf0a moved the file to the correct directory according to the new file structure and added Professor.java to the `edu.uga.devdogs.professor_rating.webscraping` package.
